src/hypergraph/viz/assets/components.js

```
@@ -277,12 +281,163 @@
+
+        var r = Math.max(0, radius || 0);
+        var path = 'M ' + pts[0].x + ' ' + pts[0].y;
+
+        if (r <= 0 || pts.length < 3) {
+          for (var i = 1; i < pts.length; i += 1) {
+            path += ' L ' + pts[i].x + ' ' + pts[i].y;
+          }
+          return path;
+        }
+
+        for (var i = 1; i < pts.length - 1; i += 1) {
```

Biome (2.3.13)

[error] Shouldn't redeclare 'i'. Consider to delete it or rename it.

'i' is defined here:


(lint/suspicious/noRedeclare)